releasenotes/notes/dbapi-rowcount-3cf18b68114a6cad.yaml

```
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    dbapi: The dbapi integration no longer assumes that a cursor object will
+      have a rowcount but not all database drivers implement rowcount.
```

```suggestion
    have a rowcount but not all database drivers implement rowcount.
```
The spacing on this line is weird. Not sure if it will cause formatting errors in the release note. If not, feel free to resolve this comment.